dev-util/capstone/capstone-4.0.1.recipe

```diff
@@ -45,10 +45,16 @@ BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
 	cmd:awk
+	cmd:find
```

Comment:
cmd:find and cmd:unzip should be moved to TEST_REQUIRES as they are not needed for the build

Reply:
Done.